SmartDeviceLink/SDLGetVehicleData.m

```diff
@@ -69,6 +98,47 @@ - (instancetype)initWithAccelerationPedalPosition:(BOOL)accelerationPedalPositio
     return self;
 }
 
+- (instancetype)initWithAccelerationPedalPosition:(BOOL)accelerationPedalPosition airbagStatus:(BOOL)airbagStatus beltStatus:(BOOL)beltStatus bodyInformation:(BOOL)bodyInformation cloudAppVehicleID:(BOOL)cloudAppVehicleID clusterModeStatus:(BOOL)clusterModeStatus deviceStatus:(BOOL)deviceStatus driverBraking:(BOOL)driverBraking eCallInfo:(BOOL)eCallInfo electronicParkBrakeStatus:(BOOL)electronicParkBrakeStatus emergencyEvent:(BOOL)emergencyEvent engineOilLife:(BOOL)engineOilLife engineTorque:(BOOL)engineTorque externalTemperature:(BOOL)externalTemperature fuelLevel:(BOOL)fuelLevel fuelLevelState:(BOOL)fuelLevelState fuelRange:(BOOL)fuelRange gps:(BOOL)gps headLampStatus:(BOOL)headLampStatus instantFuelConsumption:(BOOL)instantFuelConsumption myKey:(BOOL)myKey odometer:(BOOL)odometer prndl:(BOOL)prndl rpm:(BOOL)rpm speed:(BOOL)speed steeringWheelAngle:(BOOL)steeringWheelAngle tirePressure:(BOOL)tirePressure turnSignal:(BOOL)turnSignal vin:(BOOL)vin wiperStatus:(BOOL)wiperStatus handsOffSteering:(BOOL)handsOffSteering {
```

Comment:
I am afraid I did not get the idea, all the arguments in the initializers are BOOLs and as such they cannot be nullable, just in case I declared initializers nullable like this: -(nullable instancetype)initWithBool:(BOOL)boolValue;

Reply:
I have added the initializer generated with the RPC generator (the version from git@github.com:LuxoftSDL/rpc_spec.git; branch: /feature/0257_vehicle_data_handsOffSteering);

Should I reorder all arguments alphabetically or RPC generated methods take precedence ?

Reply:
Thanks. How does the RPC generator order them? Based on the order in the RPC spec? If so, I think it's best to use the generated version.

Reply:
It looks like the script keeps the RPC spec (MOBILE_API.xml) order. Just in case I commited both versions (hand made and script generated) could you please review and advice which one we should use? Thank you.

Reply:
As much as I don't like it what the generated code is, part of the RPC generator proposal was to use the generated types, so that is the one that should be used.